### PR TITLE
Update kubekins image in all scalability jobs.

### PR DESCRIPTION
**Pull request by mm4tt**
Looks like I missed few jobs last time.

Ref. https://github.com/kubernetes/kubernetes/issues/73105

### CODE REVIEW

**Comment by mm4tt** [comment]
/hold 
I messed up something when updating generated-security-jobs.yaml

File: config/jobs/kubernetes-security/generated-security-jobs.yaml [hunk 1]
```diff
@@ -1,3850 +0,0 @@
-# Autogenerated by genjobs.go, do NOT edit!
```
Comment (wojtek-t) [inline config/jobs/kubernetes-security/generated-security-jobs.yaml]: Looks like something is wrong with this PR - this file shouldn't be removed...

**Comment by mm4tt** [comment]
/hold cancel

Now should be good

File: config/jobs/kubernetes-security/generated-security-jobs.yaml (hunk 1 above)
Comment (mm4tt) [reply config/jobs/kubernetes-security/generated-security-jobs.yaml]: Yeah, I didn't realize that /hack/update-config.sh failed and only removed the file without writing an update one. Now should be good

**Pull request merged**